### PR TITLE
chore: remove filterExtern flag

### LINKED ISSUE #1708: Cleanup FilterExtern Flag Code & FilteredExtern Tests
- Remove FilterExterns
- Remove tests which start with `FilteredExternXYZ`/`filteredExternXYZ` and move their contents to their corresponding XYZ tests

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #1708

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3284)

File: src/shared/apis/filteredExternQuery.test.ts [hunk 1]
```diff
@@ -1,300 +0,0 @@
-import {mocked} from 'ts-jest/utils'
```
Comment (asalem1) [inline src/shared/apis/filteredExternQuery.test.ts:1]: why are we removing this test?

File: src/shared/utils/filteredExternFilterUnusedVars.test.ts [hunk 2]
```diff
@@ -1,58 +0,0 @@
-import {getMockedParse} from 'src/shared/utils/mocks/mockedParse'
-import {
```
Comment (asalem1) [inline src/shared/utils/filteredExternFilterUnusedVars.test.ts:2]: does this function now no longer exist?

File: src/shared/apis/filteredExternQuery.test.ts (hunk 1 above)
Comment (subirjolly) [reply src/shared/apis/filteredExternQuery.test.ts:1]: all the `filteredExternXXXX.test.ts` were copy of the original tests intended to run with `filterExtern` flag turned on. So, I have removed these tests and copied their contents to original tests.

File: src/shared/utils/filteredExternFilterUnusedVars.test.ts (hunk 2 above)
Comment (subirjolly) [reply src/shared/utils/filteredExternFilterUnusedVars.test.ts:2]: same as above

**Review by asalem1 — APPROVED** [review]

**Pull request merged**